### ISSUE TITLE
composer.json

### ISSUE THREAD

**Issue by mirko-pagliai** [icings/partitionable#26]
Is it possible to update the `composer.json`? It still refers to the `5.x-dev` version.

```bash
$ composer require icings/partitionable
./composer.json has been updated
Running composer update icings/partitionable
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - icings/partitionable[0.1.0, ..., 0.2.0, 1.0.0] require cakephp/orm ^4.1 -> found cakephp/orm[4.1.0, ..., 4.5.5] but these were not loaded, likely because it conflicts with another require.
    - icings/partitionable 2.0.0 requires cakephp/orm 5.x-dev -> found cakephp/orm[5.x-dev] but it does not match your minimum-stability.
    - Root composer.json requires icings/partitionable * -> satisfiable by icings/partitionable[0.1.0, 0.2.0, 1.0.0, 2.0.0].

You can also try re-running composer require with an explicit version constraint, e.g. "composer require icings/partitionable:*" to figure out if any version is installable, or "composer require icings/partitionable:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

**Comment by ndm2** [comment]
Sure, I'll take care of it later today.

**Comment by ndm2** [comment]
Fixed in [2.0.1](https://github.com/icings/partitionable/releases/tag/2.0.1)
